inspectit-oce-core/src/test/java/rocks/inspectit/oce/core/instrumentation/injection/ClassInjectorUnitTest.java

```diff
@@ -0,0 +1,203 @@
+package rocks.inspectit.oce.core.instrumentation.injection;
+
+import javassist.ClassClassPath;
+import javassist.ClassPool;
+import javassist.CtClass;
+import javassist.Modifier;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import rocks.inspectit.oce.core.testutils.DummyClassLoader;
+import rocks.inspectit.oce.core.testutils.GcUtils;
+
+import java.lang.instrument.Instrumentation;
+import java.lang.ref.WeakReference;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+public class ClassInjectorUnitTest {
+
+    @Mock
+    Instrumentation instrumentation;
+
+    @InjectMocks
+    ClassInjector injector = new ClassInjector();
```

Comment:
do you need to initialize with @InjectMock?

Reply:
You are right, it works without initialization